cmd/parse_hcl.go

```diff
@@ -90,7 +90,7 @@ func decodeHcl(
 		}
 	}
 
-	evalContext, err := config.CreateTerragruntEvalContext(filename, terragruntOptions, extensions)
+	evalContext, err := extensions.CreateTerragruntEvalContext(filename, terragruntOptions)
```

Comment:
They made CreateTerragruntEvalContext a method of config.EvalContextExtensions in v0.53.4. I guess it works better than passing these extensions as a parameter.